src/application/deployer/index.go

```diff
@@ -97,19 +104,19 @@ func validateConfigs(deployConfig types.DeployConfig) error {
 	return nil
 }
 
-func deployApplication(deployConfig types.DeployConfig, imagesMap map[string]string) error {
+func deployApplication(deployConfig types.DeployConfig, imagesMap map[string]string, secrets types.Secrets) error {
 	fmt.Fprintln(deployConfig.Writer, "Retrieving remote state...")
 	err := terraform.RunInit(deployConfig)
 	if err != nil {
 		return err
 	}
 
-	fmt.Fprintln(deployConfig.Writer, "Retrieving secrets...")
-	secrets, err := aws.ReadSecrets(deployConfig.AwsConfig)
+	fmt.Fprintln(deployConfig.Writer, "Applying changes...")
+	err = terraform.RunApply(deployConfig, secrets, imagesMap, deployConfig.DeployServicesOnly)
 	if err != nil {
-		return err
+		if strings.Contains(err.Error(), "exit status") {
```

Comment:
Not sure what the best thing to do here is. Trying to catch the error that is thrown and the apply is cancelled, but I'm thinking there could be other errors in which "exit status" is a string.

Reply:
Since we already output everything, I think this is good